### ISSUE TITLE
Removing Recipe with multiple Inputs in XML file

### ISSUE THREAD

**Issue by Freebi85** [al132mc/alchemistry#14]
Hello =)

There is a recipe to create (amongst other things) Aluminium with 4 pieces of Lapis. I try to remove this recipe but it dosent work. I think i have to declare the ammount of the input slot. But how :D 
Recipes without multiple item inputs works fine to remove.

Freebi

**Comment by al132mc** [comment]
Sorry that it took me so long (literally months!) to get back to you. I have been away from minecraft for a while. Are you still having troubles with this?

**Comment by Freebi85** [comment]
hello =)
Yes, I'm afraid I still haven't made it. I could do a lot with xml files and most of it worked. But something I do wrong with multiple inputs. I'm sure it works, but I can't find the right code for it.

**Comment by al132mc** [comment]
Alright, I took a look and it seems like it was broken. This should be fixed in the upcoming version (v0.7.16).

```
<recipe type="dissolver" action="remove">
    <input meta="4" quantity="4">minecraft:dye</input>
</recipe>
```

This is what you can use once the new version is released

**Comment by al132mc** [comment]
This has been fixed in v0.8.0 which I have just released (ignore the version I mentioned in the previous comment). https://minecraft.curseforge.com/projects/alchemistry/files/2677325

I also have crafttweaker support in this new version, so in the future you may prefer to use that anyway

**Comment by Freebi85** [comment]
Great! Thank you very much!
